Memory_Excercises/Excersice10/Source.cpp

```diff
@@ -0,0 +1,12 @@
+#include "ClassA.h"
+#include "ClassB.h"
+#include <iostream>
+
+int main()
+{
+	FClassA ClassA;
+	FClassB ClassB;
+
+	std::cout << "ans 10: the program doesn't work because the Main is calling the header file of both classes so it's making a cycle" << std::endl;
+	std::cout << "ans 11:The Forwards declarations are so useful when we need to reduce the debug's time furthermore whether we want to is so useful when we need to declare as a pointer";
```

Comment:
How are they actually helping to reduce debug time?

Reply:
x2, doesn't seem like forward declarations are very clear here hehe